### ISSUE TITLE
[BarGraph] Cannot use keyboard to show number tooltips in bar graphs

### ISSUE THREAD

**Issue by jaimiegorman** [carbon-design-system/carbon#7670]
## Environment

Test Tool Type: Manual test (AVT2)
Browser: Firefox 83.0 (64-bit)

## Detailed description

> What version of the Carbon Design System are you using?
    "carbon-components": "~10.25.0",
    "carbon-components-react": "~7.25.0",
> What did you expect to happen? Expected to be able to use a keyboard to navigate to the number tooltips 

> What happened instead? Keyboard passes by numbers 

> What WCAG 2.1 checkpoint does the issue violate? IBM CI 162 Checkpoint Violation: 2.1.1 Keyboard
IBM Guidance:
https://www.ibm.com/able/guidelines/ci162/keyboard.html

## Steps to reproduce the issue

1. Login to RTWA employer home page: https://test.wh-rtw.dev.watson-health.ibm.com/analytics/
2. Access RTWA employer homepage
3. Use keyboard to select a workplace in Search bar of the Exploring and comparing workplaces tab
4. Use keyboard to navigate to Status change chart
5. Use keyboard to navigate to Workforce status chart in Understanding a workplace's conditions

> Please create a reduced test case in CodeSandbox
>
> - Style and vanilla JS:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/components/examples/codesandbox
> - React:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/codesandbox

## Additional information

- Screenshots or code
- Notes
![image](https://user-images.githubusercontent.com/78103524/106039343-06833880-60a7-11eb-8b07-03bd25edc2cf.png)
![image](https://user-images.githubusercontent.com/78103524/106039363-097e2900-60a7-11eb-88af-e041428a801f.png)
